APCAppCore/APCAppCore/DataSubstrate/Model/APCUser+Bridge.m

```diff
@@ -75,6 +75,37 @@ - (void)signUpOnCompletion:(void (^)(NSError *))completionBlock
     }
 }
 
+- (void)signUpWithDataGroups:(NSArray<NSString *> *)dataGroups onCompletion:(void (^)(NSError *))completionBlock
```

Comment:
You can go ahead and just have signUpOnCompletion: call this method with nil for the dataGroups argument rather than having duplicated code, since that's what the SDK does internally anyway.